corehq/apps/api/tests/test_case_update_api.py

```diff
@@ -0,0 +1,179 @@
+import uuid
+
+from django.test import TestCase
+
+from casexml.apps.case.mock import CaseBlock
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.apps.users.models import WebUser
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors, FormAccessors
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+
+
+class TestUpdateCases(TestCase):
+    domain = 'test_update_cases'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+        cls.web_user = WebUser.create(cls.domain, 'netflix', 'password', None, None)
+        cls.case_accessor = CaseAccessors(cls.domain)
+        cls.form_accessor = FormAccessors(cls.domain)
+
+    def tearDown(self):
+        FormProcessorTestUtils.delete_all_cases(self.domain)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
+        super().tearDownClass()
+
+    def post(self, resource_url, body):
+        # TODO
+        pass
+
+    def test_create_case(self):
+        res = self.post('create_case', {
+            # notable exclusions: case_id, date_opened, date_modified
```

Comment:
The plan is to only set case_ids on the server side, to avoid any issues with partners not using proper generated UUIDs.